src/screens/Episode/components/Player/Player.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useState } from 'react'
+import { View, Pressable } from 'react-native'
+import TrackPlayer, { useProgress, Capability } from 'react-native-track-player'
+import Slider from '@react-native-community/slider'
+import { getTime } from 'src/screens/Episode/helpers'
```

Comment:
It looks wrong, that you import Episode helper in Player. Either put it to `/src/helpers` or `/Player/common/helpers`